pkg/platform/platform.go

```diff
@@ -140,6 +142,15 @@ func InfraStatus(client cnoclient.Client) (*bootstrap.InfraStatus, error) {
 		res.MasterIPsecMachineConfig = masterIPsecMachineConfig
 	}
 
+	// Retrieve master's IPsecMachineConfig if it's created with different name.
+	if res.MasterIPsecMachineConfig == nil {
+		masterIPsecMachineConfig, err = getIPsecMachineConfig(client, "machineconfiguration.openshift.io/role=master")
+		if err != nil {
+			return nil, fmt.Errorf("failed to get ipsec machine config for master: %v", err)
+		}
+		res.MasterIPsecMachineConfig = masterIPsecMachineConfig
+	}
+
```

Comment:
Why do you need this? You just added the Machine configs in this PR. Please, explain and split on its own commit because I don't see how this is related to the commit you added it.

Reply:
added with commit 239d296741c5f329b05d83f055c964f24d16ec27 and doc comment in the code.